examples/demo/Extras/Tree_editor_with_TreeNodeRenderer.py

```diff
@@ -14,17 +14,14 @@
 contents inside the cells of a TreeEditor.
 """
 
-    absolute_import, division, print_function, unicode_literals
```

Comment:
Err. What happened here?

Reply:
A bad search-replace for `from __future__` in #767

Reply:
And the reviewer didn't catch it. For shame ...